src/lang/en.json

```
@@ -731,6 +731,7 @@
     "Icon Emoji": "Icon Emoji",
     "signalImportant": "IMPORTANT: You cannot mix groups and numbers in recipients!",
     "aboutWebhooks": "More info about Webhooks on: {0}",
+    "aboutUsername": "The {0} {1} can change the display name of the message sender. This is not for mentions. If you want to mention someone, include it in the {2}.",
```

lets reword it like this, what do you think?

```suggestion
    "aboutSlackUsername": "Changes the display name of the message sender. If you want to mention someone, include it in the friendly name instead.",
```